images/bootstrap/Dockerfile

```diff
@@ -90,6 +90,15 @@ RUN if test "${ARCH}" != s390x; then \
         rm -rf /root/.cache/bazel; \
     fi
 
+# As gcloud isn't supported on s390x, rather than installing kubectl via gcloud as in above RUN,
+# we are installing  its binary as described in the kubernetes official docs.
+RUN if test "${ARCH}" == s390x; then \
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/s390x/kubectl" && \
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl && \
```

Comment:
Is `sudo` required here? I am not sure it is as we are using dnf above without sudo.

Reply:
It seems sudo is not required for any commands inside Dockerfile as root is default user, unless someone changes and I don't see we are changing. So removed sudo. Thanks for pointing it.